protocol/database-scripting/README.md

```diff
@@ -0,0 +1,102 @@
+Database Scripting
+==============
+
+A protocol for running console scripts on the Healthify app's production database.
+
+Development
+-----------
+
+Develop the scripting story similarly to any other story. In this case, this would
+entail running your script at the rails console of your local server.
+
+While some scripts only lend themselves to being performed via pure SQL, where it
+is possible, you should leverage ActiveRecord to perform our INSERT/UPDATE/DELETE
+operations so that we leverage ActiveRecord validations and callbacks.
+
+While developing the script, it is critical to plan our remediation strategy if
+something does go awry.
```

Comment:
To be honest, I'm not up-to-date with the state of our logging, so I'm not sure how you would go about doing this. Perhaps you or @switzersc could fill me in on how we could employ such a tactic.